introduction.md

```diff
@@ -10,13 +10,13 @@ position: 1
 
 Thank you for choosing Telerik® UI for Blazor!
 
-The [Blazor framework](https://blazor.net/) by Microsoft allows you to create web applications with .NET and C# to create front-end. The Telerik® UI for Blazor components facilitate the front-end development by providing you with ready made UI components.
+The <a href="https://blazor.net/" target="_blank">Blazor framework by Microsoft</a> allows you to create rich web UIs by using .NET and C#. The Telerik® UI for Blazor components facilitate the front-end development by providing ready-made UI components.
 
->tip Telerik UI for Blazor supports both Client-side (WebAssembly) Blazor and Server-side Blazor apps.
+>tip Telerik UI for Blazor supports both client-side (WebAssembly) and server-side Blazor apps.
 
-The UI for Blazor suite has the same HTML rendering and Theme stylesheets like other Kendo UI suites, so previous experience with them can be helpful. The components in **UI for Blazor are native components** and not wrappers over jQuery widgets, however.
```

Comment:
For emphasis, use italic instead of bold. See, https://testdocs.telerik.com/style-guide/element-formatting